guides/INSTALLATION.md

```
@@ -0,0 +1,571 @@
+
+### macOS
+
+#### 问题: "无法打开" 或 "已损坏"
+
+**解决方法 1**: 右键打开
+1. 右键点击应用
+2. 选择 **"打开"**
+3. 在弹出的对话框中点击 **"打开"**
+
+**解决方法 2**: 移除隔离属性
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)